packages/react-native/README.md

```diff
@@ -0,0 +1,56 @@
+# PubNub Chat Components for React Native
+
+PubNub Chat Components for React Native are the fastest way to add chat features like direct and group messaging, typing indicators, reactions, without going through the complexity of low-level architecture of realtime networks.
+
+- **Reduced Implementation Time**: Develop proof-of-concept and production ready apps faster using predefined components.
+- **Fast and Simple Extensibility**: Add rich features like typing indicators, read receipts, reactions etc. without writing complex code.
+- **Flexible and Customizable Components**: Customize component design and add custom components to extend functionality.
+- **High Scalability**: Let PubNub take care of scaling and reliability as you grow your app.
+- **Easy Theming**: Use the built-in light and dark themes or create custom ones for various use cases: group, support, and event chats.
+- **Strong Typing**: Utilize the power of TypeScript to develop your application.
+
+> You can also check out our [PubNub Chat Components for React](https://www.npmjs.com/package/@pubnub/react-chat-components)
```

Comment:
I wanted to have a cross-link between packages on npm so people are aware of the existence of separate packages before they land on our docs. Wondering if there's better wording for this